apps/iglidur-designer/src/app/libs/header/src/lib/languages/languages.component.ts

```diff
@@ -11,11 +12,22 @@ import { findFlagUrlByCountryName } from 'country-flags-svg';
   styleUrl: './languages.component.scss',
 })
 export class LanguagesComponent {
+  constructor(private languageService: LanguageService) {}
+  @Output() emitClose = new EventEmitter<boolean>();
```

Comment:
I see you are using the new control flow. do you know about the new input and output?

Reply:
I've refactored to use the new output